guidelines/mig-drd.md

```diff
@@ -277,11 +277,16 @@ TBD
 ### [REQ-DCW-06] MIG SHALL be able to allow users to upload DICOM objects via DICOMweb STOW-RS
```

Comment:
Will you add a new requirement for allowing submission of a job to MONAI Deploy via STOW-RS using a REST call ?

Reply:
I was assuming it was over REST. If not, what other mechanism does STOW-RS use?

Reply:
`REQ-FNC-02` covers all methods of triggering workflows, e.g. C-STORE, ACR, and in this case, STOW-RS>